src/test/java/uk/gov/hmcts/reform/pdf/service/client/PDFServiceClientTest.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmcts.reform.pdf.service.client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.RestOperations;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class PDFServiceClientTest {
+
+    private static final String API_MIME_TYPE = "application/vnd.uk.gov.hmcts.pdf-service.v2+json";
```

Comment:
I meant to extract this so that tests and lib use the same constant :)

Reply:
Then we would be testing if implementation refers to the same constant as tests do. I think it's safe enough to leave it as is.

Reply:
You could say the same about `MediaType.APPLICATION_PDF`...

There should be one 'source of truth' for what the pdf service mimetype is.

I suggest adding a public constant to PdfClient class and using it here in tests.

Reply:
As this mime type is defined in our application, I think it's better as it is. If someone changes the value by accident, at least tests will start failing.

Reply:
Done